client/src/app/components/target-card.tsx

```diff
@@ -137,7 +138,9 @@ export const TargetCard: React.FC<TargetCardProps> = ({
                   </DropdownItem>,
                 ]}
               />
-            ) : null}
+            ) : (
+              readOnly && item.custom && <Label color="grey">Custom</Label>
```

Comment:
`readOnly` test is not needed, only `item.custom`.

And the true case first would help make it easier to read:
```ts
<FlexItem className={spacing.mlAuto}>
  {readOnly
    ? item.custom && <Label color="grey">Custom</Label>
    : item.custom && (
        <KebabDropdown
          dropdownItems={[
            <DropdownItem key="edit-custom-card" onClick={onEdit}>
              {t("actions.edit")}
            </DropdownItem>,
            <DropdownItem key="delete-custom-card" onClick={onDelete}>
              {t("actions.delete")}
            </DropdownItem>,
          ]}
        />
      )}
</FlexItem>
```

Reply:
Good idea - Fixed.